workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/utils/constants/filters.ts

```diff
@@ -76,7 +77,7 @@ export const Filters = {
 	},
 	DEAL_LISTING: {
 		rejected: `${fiscalYearFilterCreatedDate} and leadType eq 'Partner Qualified Lead (PQL)' and leadStatus eq 'CAM rejected'`,
-		submitted: `createdDate ge 2023-01-01T00:00:00Z and leadType eq 'Partner Qualified Lead (PQL)' and isConverted eq false and leadStatus ne 'CAM rejected'`,
+		submitted: `createdDate ge ${previousFiscalYearStart}T00:00:00Z and leadType eq 'Partner Qualified Lead (PQL)' and isConverted eq false and leadStatus ne 'CAM rejected'`,
```

Comment:
this should always be 2023-01-01 and not previousFiscalYear, you can create a const to store the date value, and use it.

Reply:
FYI https://liferay.atlassian.net/wiki/spaces/~713302437/pages/2544631931/PRM+PROD+Dashboard+and+Listing+Data+Mastersheet

Reply:
In the development conversations I had understood that it was to exchange this string data for an equivalent variable. I believe there was a failure communication. I will see the documentation, thanks for your attention.